lib/commands/utility.go

```diff
@@ -1,19 +1,189 @@
 package commands
 
-import "github.com/urfave/cli/v2"
+import (
+	"fmt"
+
+	"github.com/fatih/color"
+	"github.com/kionsoftware/kion-cli/lib/helper"
+	"github.com/kionsoftware/kion-cli/lib/kion"
+	"github.com/kionsoftware/kion-cli/lib/structs"
+	"github.com/urfave/cli/v2"
+)
 
 // FlushCache clears the Kion CLI cache.
 func (c *Cmd) FlushCache(cCtx *cli.Context) error {
 	return c.cache.FlushCache()
 }
 
 // PushFavorites pushes the local favorites to a target instance of Kion.
-func (c *Cmd) PushFavorites(cCtx *cli.Context) error {
-	// identify target instance of Kion
-	// check version of target instance of Kion
-	// pull existing upstream favorites
-	// deconflict with local version of favorites, id to be pushed
-	// prompt user to confirm what will be pushed where
-	// perform push or bail
-	return nil
+func (c *Cmd) PushFavorites(configPath string) func(cCtx *cli.Context) error {
```

Comment:
What do you think about putting the config path into the context metadata like we do with the is3.13.0 bool feature flag so that we can simplify the func signature and keep it consistent with the other commands?

Reply:
Yea, sounds good. I'll get that in.